lunatrace/bsl/frontend/src/layouts/Main.tsx

```diff
@@ -61,19 +62,22 @@ const MainLayout: React.FunctionComponent = (props) => {
 
   const [trigger, result, lastPromiseInfo] = api.useLazyGetSidebarInfoQuery();
   const { data } = result;
+
   useEffect(() => {
     if (isAuthenticated) {
       void trigger();
     }
   }, [isAuthenticated]);
 
+  const setupWizardOpen = !userHasAnyOrganizations(data);
+
   return (
     <React.Fragment>
       <Wrapper>
         <Sidebar sections={generateSidebarItems(data, isAuthenticated)} />
         <div className="main">
-          <Navbar />
-          <NavbarBreadcrumbs />
+          <Navbar setupWizardOpen={setupWizardOpen} />
+          <NavbarBreadcrumbs setupWizardOpen={setupWizardOpen} />
```

Comment:
For the breadcrumbs, since this just returns `null` if `setupWizardOpen === true` internally, I would just skip rendering it in `Main.tsx` instead. That is simpler to understand at a glance without needing to go read the source code of `NavbarBreadcrumbs` to understand it's effect.